### ISSUE TITLE
RFE: do/until language feature

### ISSUE THREAD

**Issue by mpdehaan** [ansible/ansible#2959]
We want to be able to enable retry logic.   Example:

```
- action: shell /usr/bin/foo
  register: result
  until: register.stdout.find("all systems go") != -1
  retries: 5
  delay: 10
```

**Comment by srgvg** [comment]
definitely +1

**Comment by bcoca** [comment]
Would love this!
default for until is 'ok' ?

**Comment by mpdehaan** [comment]
this has been implemented and is now in docs for 1.4, so closing.
